### PR TITLE
Add helmfile/terraform runner with hierarchical configuration support

### LINKED ISSUE #43: Add helmfile/terraform runner with hierarchical configuration support

### PR DESCRIPTION
**Pull request by costimuraru**
fixes #43 

```
ops data/env=dev/region=va6/project=ee/cluster=experiments terraform plan
ops data/env=dev/region=va6/project=ee/cluster=experiments helmfile sync

ops data/env=dev/region=va6/project=ee/cluster=experiments config --format json --output-path path/to/file.json
```

### CODE REVIEW

File: src/ops/ee/composition_config_generator.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 Adobe. All rights reserved.
```
Comment (amuraru) [inline src/ops/ee/composition_config_generator.py]: @costimuraru WOuld it make sense to move `src/ops/ee` to `src/ops/hierarchical` package ?

**Comment by amuraru** [comment]
@costimuraru this looks awesome.
Couple of things we may want in this PR:
1. Add a simple hierarchical configuration example, like in your examples:
`data/env=dev/region=va6/project=myProject/cluster=dev1`
2. Add a new section in README.md to explain this new functionality.


Also I would also bump the major version for this tool to 1.0
and add a deprecation warning to old flat file based config?

**Comment by costimuraru** [comment]
> Add a simple hierarchical configuration example, like in your examples: data/env=dev/region=va6/project=myProject/cluster=dev1

Done

> Add a new section in README.md to explain this new functionality.

Done

> Also I would also bump the major version for this tool to 1.0

Done 

> and add a deprecation warning to old flat file based config?

I don't know about this one... do we want to impose hierarchical configs on our existing repos (cns)?

File: src/ops/ee/composition_config_generator.py (hunk 1 above)
Comment (costimuraru) [reply src/ops/ee/composition_config_generator.py]: Done

**Pull request merged**